File: FirstBitSolutions/C Module/Assignments/Assignment 6/Func_Assignment_3/FunctionsType3/Ques5.c
//5 Check the given number is Armstrong number or not..
//Input: n = 153
//Output: Armstrong
#include<stdio.h>
void Armstrong(int,int,int,int,int);//Declaration
void main()
{
	int n=153,no,rem,sum;
	int temp = sum = 0;
	Armstrong(n,no,rem,sum,temp);//Call		
}//main ends here
void Armstrong(int a,int b,int c,int d,int e){

	
	while(b>0){
		c = b%10;
		b = b / 10;
		d = d + c*c*c;

	}
	if(e == d){
		printf("This is Armstrong number");
	}else{
		printf("This is not Armstrong number");
	}
}//Armstrong ends here

File: FirstBitSolutions/C Module/CtoJava Module/Assignment 3/Ques5.java
class Ques{ 
public static void main(String args[]){ 

	int no=153;
	int rem,sum;
	int temp = sum = 0;
	
	while(no>0){
		rem = no%10;
		no = no / 10;
		sum = sum + rem*rem*rem;

	}
	if(temp == sum){
		System.out.printf("This is Armstrong number");
	}else{
		System.out.printf("This is not Armstrong number");
	}
 }
}